app/controllers/control_panel/boards_controller.rb

```diff
@@ -1,11 +1,13 @@
 class ControlPanel::BoardsController < ControlPanel::ControlPanelController
+  before_action :require_member, except: [:index, :new, :create]
+  before_action :require_owner, only: [:edit, :update, :destroy]
```

Comment:
Many people claimed that it is a anti-pattern but IMO if you use it sensible it make improve you code.